### DISCUSSION TITLE
Super Chlorinate not controlling

### DISCUSSION THREAD

**Discussion by Hect0rG** [tagyoureit/nodejs-poolController#418] [Nixie] [answered]
@rstrouse you helped me a short while ago with the super chlorinate when it wasn't working. And you were able to get it to initialize after a minor code change. This weekend I went to test it out and I noticed that although it looked like it was initializing properly, after a few seconds it goes back to whatever it was doing prior to pressing Super Chlorinate. The Cancel Super Chlorinate button goes away and it drops from 100% back to 75%/ On the chlorinator itself, I do not see the super chlorinate led come on, I imagine that it should turn on but not sure if you just ramp up the percentage to 100 and not worry about triggering the actual super Chlorinate option.

**Comment by rstrouse** [comment]
The super chlorinate or boost mode that is on the controller does not do anything special with cell power.  It only turns the power up to 100% for 24 hours then returns it back to the original setting.  You cannot set the time if using the built-in function and if it receives commands from an external controller it will pop it out of the built in function.

Can I get you to create a replay of this process when you click the Super Chlorinate button?  Start the Capture Replay then wait a couple of seconds then turn on the Super Chlorinate and wait for several seconds after the cancel button disappears.  Then go back to Cancel Replay and upload the resultant files.

**Reply by Hect0rG** [reply]
Sure Ill do that as soon as I get a chance.

**Reply by Hect0rG** [reply]
[replay(1).zip](https://github.com/tagyoureit/nodejs-poolController/files/8508809/replay.1.zip)
See attached.

**Comment by rstrouse** [comment, marked as answer]
Pull njsPC.  I found a place where the watchdog was being triggered shortly after the timer was started.  This should allow it to stay in super chlorinate mode for the entire duration.

**Reply by Hect0rG** [reply]
Sounds good Ill give it a try later today and report back.

**Comment by Hect0rG** [comment]
Gave it a try last night and it works great. I went theough a full 1 hr cycle. Thanks for taking care of it. noticed the percentage on chlorinator went to 100 while it was on now.

**Reply by rstrouse** [reply]
Awesome
